docs/product-user-guide/src/main/asciidoc/con-business-central-settings-project.adoc

```diff
@@ -0,0 +1,8 @@
+[[con-business-central-settings-project]]
+= Projects
+
+Use the Project perspective to enable or disable the GAV conflict check and allow or forbid child GAV editions. 
```

Comment:
Maybe add a sentence about what "GAV conflict check" actually does.

When enabled, this check verifies that the artifact with given GAV (Group/Artifact/Version) doesn not already exists in maven repository(ies) managed by business central. If it does, it shows a modal dialog to the user asking him if he wants to override this artifact or cancel the deployment of the artifact to the maven repository.

When disabled, there is no check and artifacts are always overwritten, even when artifact with given GAV already exists in the repository.

I'm nor sure what the other option "forbid child GAV edition" does though.